doc/03-Configuration.md

```diff
@@ -128,6 +128,14 @@ the cluster nodes resynchronise their states after each executed query, and may
 However, this does not necessarily have to be the case if, for instance, Icinga DB is only allowed to connect to a
 single cluster node at a time. This is the case when a load balancer does not randomly route connections to all the
 nodes evenly, but always to the same node until it fails, or if your database cluster nodes have a virtual IP address
-fail over assigned. In such situations, you can set the `wsrep_sync_wait` system variable to `0` in the 
+fail over assigned. In such situations, you can set the `wsrep_sync_wait` system variable to `0` in the
 `/etc/icingadb/config.yml` file to disable it entirely, as Icinga DB doesn't have to wait for cluster
 synchronisation then.
+
+### Systemd Journald Fields
+
+When examining the journal with `journalctl`, fields containing additional information are hidden by default.
+Setting an appropriate
+[`--output` option](https://www.freedesktop.org/software/systemd/man/latest/journalctl.html#Output%20Options)
+will include them, such as: `journalctl --output verbose` or `journalctl --output json`.
```

Comment:
I would also add `--unit 'icingadb.service'` to those example commands given that this is all about Icinga DB.

Reply:
I get your point. However, including the `--unit` argument for both examples makes this sentence very long, hiding the highlighted `--output` argument.

> Setting an appropriate [--output option](https://www.freedesktop.org/software/systemd/man/latest/journalctl.html#Output%20Options) will include them, such as: `journalctl --unit icingadb.service --output verbose` or `journalctl --unit icingadb.service --output json`.

Thus, I would prefer to drop `journalctl` and just show the two `--output` arguments.


> Setting an appropriate [--output option](https://www.freedesktop.org/software/systemd/man/latest/journalctl.html#Output%20Options) will include them, such as: `--output verbose` or `--output json`.

Reply:
> However, including the `--unit` argument for both examples makes this sentence very long, hiding the highlighted `--output` argument.

But I don't quite get your point. Isn't this docs snippet supposed to help someone find the Icinga DB fields? I mean, OK, you want to highlight the `--output` argument even more, but in the end the user just needs a copy-pasteable one without having to construct the full command on his own. Personally, this just doesn't seem like a good approach to only partially document something.

Reply:
My assumption is that users have a basic understanding of journalctl, as otherwise they simply cannot operate it. Using it without any further parameter would result in logs since last log vacuum, which might have never happened. Thus, they might want to use `-f` or `-n 100` or `--since today` or something else..

I don't see any one-command-fits-it-all. My intention was just to point out that the necessary details are hidden, which is a non-default behavior, compared to most other software out there.

I have added a line showing one generic copy-pastable journalctl command.